packages/deployment/db/config.yml

```diff
@@ -0,0 +1,30 @@
+default:
```

Comment:
I'm a bit confused and I think I'm misunderstanding something. I thought we were going to have a single config file and get away from individual config files? Or are we still internally going to have individual config files for stacks but allow users to manage/override all the config in a single file?

Reply:
One user-facing config file. The deployment package will have separate config files because of the way kes handles kes folders. An alternative would be to pass files in without using kes-folder and using the individual file options of kes, e.g. `--cfFile`, but I wanted to limit the number of user-facing changes. I'm open to it if you feel strongly that it should be done.